auditlog/conf.py

```diff
@@ -15,3 +15,8 @@
 settings.AUDITLOG_INCLUDE_TRACKING_MODELS = getattr(
     settings, "AUDITLOG_INCLUDE_TRACKING_MODELS", ()
 )
+
+# Disable on raw save to avoid logging imports and similar
+settings.AUDITLOG_DISABLE_ON_RAW_SAVE = getattr(
+    settings, "AUDITLOG_DISABLE_ON_RAW_SAVE", False
```

Comment:
Isn't it a change in the current behavior? Currently django-auditlog logs the import.
I would say the default value should be `True` to keep the old behavior and don't break it